modules/ROOT/pages/upgrade.adoc

```diff
@@ -71,17 +75,17 @@ the cluster is updating in background
 $ sudo gravity enter
 $ kubectl get pods
 ----
-+
-* Login to the Ops Center at the following URL:
+
+. Log in to the Ops Center at the following URL:
 +
 ----
 https://{platform-host}:9500
 ----
 +
 [WARNING]
-Before accessing Anypoint Platform PCE, ensure that you configure your NSF servers described in the following step. Failure to configure NFS before accessing the platform can result in data loss.
+If you are upgrading from Anypoint Platform PCE 1.7.x to 2.0.x, ensure that you configure your NSF servers described in the following step before accessing Anypoint Platform PCE. In this case, failure to configure NFS before accessing the platform can result in data loss.
 
-. After upgrading Anypoint Platform PCE, configure it to use the required NFS server.
+. If you are upgrading from from Anypoint Platform PCE 1.7.x to 2.0.x, configure the required NFS server.
```

Comment:
Shouldn't they have done this already? Isn't this part of the prerequisites? We should not be documenting this over and over again.
```suggestion
. If you are upgrading from from Anypoint Platform PCE 1.7.x to 2.0.x, configure the required NFS server:
```